climada/engine/unsequa/calc_base.py

```diff
@@ -500,10 +536,44 @@ def _calc_sens_df(method, problem_sa, sensitivity_kwargs, param_labels, X, unc_d
         else:
             sens_indices = method.analyze(problem_sa, Y,
                                                     **sensitivity_kwargs)
+        #refactor incoherent SALib output
+        nparams = len(param_labels)
+        if method.__name__ == 'SALib.analyze.ff':
+            if sensitivity_kwargs['second_order']:
+                #parse interaction terms of sens_indices to a square matrix
+                #to ensure consistency with unsequa
+                interaction_names = sens_indices.pop('interaction_names')
+                interactions = np.full((nparams, nparams), np.nan)
+                for i,ie in enumerate(interaction_names):
```

Comment:
I think you could use clearer variable names. What does ie mean? What does 'IE' mean?

Reply:
I changed 'ie' to 'interaction_name'. 'IE' actually stands for 'Interaction Effect'. As it is the actual name of the indice for the 'ff' method from SALib I would keep it this way, what do you think?